Require explicit `cacheLife` on outer `"use cache"` when nesting short-lived caches (#89481)
vercel/next.js: docs/01-app/01-getting-started/06-cache-components.mdx

}
```

> > **Good to know:** Short-lived caches with `revalidate: 0` or `expire` under 5 minutes are automatically excluded from prerenders and become dynamic holes instead. This includes the `seconds` profile. If such a cache is nested inside another `use cache` without an explicit `cacheLife`, Next.js will throw an error during prerendering to prevent accidental misconfigurations. See [Prerendering behavior](/docs/app/api-reference/functions/cacheLife#prerendering-behavior) for details.

Reviewer: ```suggestion
> **Good to know:** A cache is considered "short-lived" when it uses the `seconds` profile, `revalidate: 0`, or `expire` under 5 minutes. Short-lived caches are automatically excluded from prerenders and become dynamic holes instead. If such a cache is nested inside another `use cache` without an explicit `cacheLife`, Next.js will throw an error during prerendering to prevent accidental misconfigurations. See [Prerendering behavior](/docs/app/api-reference/functions/cacheLife#prerendering-behavior) for details.
```
